src/CaptureEventProducer/CaptureEventProducer.cpp

```diff
@@ -109,6 +110,7 @@ bool CaptureEventProducer::NotifyAllEventsSent() {
 
 void CaptureEventProducer::ConnectAndReceiveCommandsThread() {
   ORBIT_CHECK(producer_side_service_stub_ != nullptr);
+  orbit_base::SetCurrentThreadName("ConnectAndReceiveCommandsThread");
```

Comment:
nit. On Linux, the maximum thread name length is 16 including the terminating null byte. Consider making is smaller, "ConnectRcvCmds" maybe?

Reply:
Yep. It's not nitpicking. The actual name you are giving here is "ConnectAndRecei".

Reply:
Hmm, the other threads I deal with are "default-executo", "resolver-execut", "grpc_global_tim" so I also just ignored the issue. Since we have the freedom to select something I went with Pierric's suggestion.